docs/source/examples/notebooks/solvers/dae-solver.ipynb

```diff
@@ -297,7 +297,7 @@
     "disc.process_model(model)\n",
     "print(f\"y0_guess={model.concatenated_initial_conditions.evaluate().flatten()}\")\n",
     "\n",
-    "dae_solver = pybamm.CasadiSolver(mode=\"safe\")\n",
+    "dae_solver = pybamm.IDAKLUSolver()\n",
```

Comment:
Can we remove these as well

Reply:
I think we should keep this one, because they intentionally use the solver interface to explain how it works under the hood